### PR TITLE
Responsive styles fixing

### PR DESCRIPTION
**Pull request by react-developer-ssa-group**

### CODE REVIEW

File: packages/widgets/src/components/Filters/FiltersBlock.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import { DropdownOption, MultipleDropdown } from '@ssa-ui-kit/core';
 import { useFiltersContext } from './FiltersContext';
+import { useTheme } from '@emotion/react';
```
Comment (arthur-kashaiev-ssa-group) [inline packages/widgets/src/components/Filters/FiltersBlock.tsx:4]: Move this higher.

**Review by arthur-kashaiev-ssa-group — CHANGES_REQUESTED** [review]

File: packages/widgets/src/components/Filters/FiltersBlock.tsx (hunk 1 above)
Comment (react-developer-ssa-group) [reply packages/widgets/src/components/Filters/FiltersBlock.tsx]: Moved

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 6       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 19  | ❗       |
    


<details>
<summary>🟡 custom/widgets-filters--filter-button-items-not-selected__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 custom/widgets-filters--more-button-items-selected__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-filters--all-items-enabled__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-filters--all-items-enabled__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-filters--all-items-enabled__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/widgets-filters--all-items-enabled__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>❗ custom/widgets-collapsiblenavbar-opened__[w1439px]_statistics_submenu_popover (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-collapsiblenavbar-opened__[w1920px]_statistics_submenu_popover (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-collapsiblenavbar-opened__[w899px]_statistics_submenu (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-filters--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-filters--more-button__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-searchbox--cross-icon__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-searchbox--search-icon__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-bots-table--bots-table__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-bots-table--bots-table__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-bots-table--no-control-orders__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-bots-table--no-control-orders__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-filters--default__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-filters--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-filters--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-filters--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-search-box--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-search-box--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-table-filters--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Pull request merged**